backends/p4tools/common/options.cpp

```diff
@@ -181,6 +181,14 @@ AbstractP4cToolOptions::AbstractP4cToolOptions(cstring message) : Options(messag
         },
         "Provides a randomization seed");
 
+    registerOption(
+        "--disable-info", nullptr,
```

Comment:
Would it make sense to include the keyword "log" or "logging" in the name of the flag somehow?

Reply:
--disable-info-logging?